src/Helpers/initialStateCity.jsx

```diff
@@ -0,0 +1,2 @@
+export const initialStateCity =
+  JSON.parse(localStorage.getItem('city')) ?? 'Minsk';
```

Comment:
move all localstorage stuff to separate util func

Reply:
fix